### DISCUSSION TITLE
issue in search box of site

### DISCUSSION THREAD

**Discussion by MohammadAsadi4** [twbs/bootstrap#39461] [Q&A]
Hi
the search box provided in bootstrap site does not work at all.what ever i searched, it does not bring any result 

The reported problem is about bootstrap site functionality and developer team know the solution

### What operating system(s) are you seeing the problem on?

Linux

### What browser(s) are you seeing the problem on?

Firefox

### What version of Bootstrap are you using?

5.3

**Comment by julien-deramond** [comment]
Hi @MohammadAsadi4 
I've moved your issue to our Q&A discussions. I'm not sure to understand what's the problem here. Could you please provide more detail regarding that?

**Comment by MohammadAsadi4** [comment]
Hi
You know that your site has  a search box in top middle of it's page
When I want to look for something I enter my phrase in that box but
whatever I entered it would not bring any result
I hope this explanation could clarify the issue

Thanks

**Reply by julien-deramond** [reply]
Can you spot any errors in the dev tools in your browser? Unreachable URLs or errors in the console?

**Comment by MohammadAsadi4** [comment]
i attached the image of dev tools
my browser is firefox and my OS is ubuntu
